torch/_functorch/aot_autograd.py

```diff
@@ -2115,6 +2136,31 @@ def compute_overlapping_inputs(fwd_inputs, aliased_input_indices):
                 actual_aliased_indices.add(j_)
     return actual_aliased_indices
 
+
+def _check_can_be_in_graph(mutates_data, mutates_metadata, requires_grad, keep_input_mutations: bool, is_train: bool):
```

Comment:
docblocks would be helpful

Reply:
rename to `_check_if_mutation_can_be_in_graph`

Reply:
The meaning of `requires_grad` is not here so make sure to xref it (requires grad here refers to the requires gradness AFTER you've run the function)